### PR TITLE
feat: [FFM-2523]: add sdk key dialog improvements

### PR DESCRIPTION
**Pull request by crystal-fong**
##### Summary:
This PR applies standardisation to the dialog when user is adding a new SDK key
- Create and Cancel buttons using Primary and Tertiary Buttons respectively
- Include a close button
- Uses Dialog component from uicore instead of blueprintjs
- Uses default styling for the header

##### Screenshots:
**Before**

image

**After**
image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by crystal-fong** [comment]
trigger cypress

**Comment by crystal-fong** [comment]
trigger cypress

File: src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx [hunk 1]
```diff
@@ -85,71 +88,53 @@ const AddKeyDialog: React.FC<Props> = ({ disabled, primary, environment, onCreat
...
                 <Button
                   text={getString('createSecretYAML.create')}
                   onClick={() => formikProps.handleSubmit()}
                   intent="primary"
                   disabled={loading}
+                  variation={ButtonVariation.PRIMARY}
+                />
+                <Button
+                  text={getString('cancel')}
+                  onClick={() => formikProps.handleReset()}
```
Comment (knagurski) [inline src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx:134]: I wonder if you can drop this and just set the type to `reset`?

File: src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx [hunk 2]
```diff
@@ -85,71 +88,53 @@ const AddKeyDialog: React.FC<Props> = ({ disabled, primary, environment, onCreat
...
+              </Text>
+              <CardSelect
+                cornerSelected
+                data={keyTypes}
+                selected={getTypeOption(formikProps.values.type)}
+                className={css.cardSelect}
+                onChange={nextValue => formikProps.setFieldValue('type', nextValue.value)}
+                renderItem={cardData => (
+                  <Container
+                    flex={{ align: 'center-center', distribution: 'space-between', justifyContent: 'center' }}
```
Comment (knagurski) [inline src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx:117]: In a `flex` prop the `align` is shorthand for both `align-items` and `justify-content`. Also `distribution` is an alias for `justify-content`. I _think_ you can get away with just the `align: 'center-center'` and yank the other two as they're kinda just stepping on each other's toes.

**Review by knagurski — APPROVED** [review]
Really solid work. Just a couple of suggestions/observations.

File: src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx (hunk 1 above)
Comment (crystal-fong) [reply src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx:134]: No I tried it :( , tried replacing `formikProps.handleSubmit` with `submit` too with no luck

Comment (crystal-fong) [reply src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx:134]: ooh actually, it'll work if I wrap it in `FormikForm` 👍

**Comment by crystal-fong** [comment]
trigger cypress

**Review by laura-mannus — APPROVED** [review]

**Pull request merged**